specification/mpcnetworkfunction/mpcnetworkfunction.Management/tspconfig.yaml

```
@@ -0,0 +1,10 @@
+output-dir: "{project-root}"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: ./swagger/MobilePacketCore/resource-manager
+    examples-directory: ./examples
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/networkfunction.json"
```

```suggestion
emit:
  - "@azure-tools/typespec-autorest"
options:
  "@azure-tools/typespec-autorest":
    azure-resource-provider-folder: "resource-manager"
    emitter-output-dir: "{project-root}/.."
    examples-directory: "examples"
    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/networkfunction.json"
```

See this spec as an example:

https://github.com/Azure/azure-rest-api-specs/blob/9025f6894e0df9ebbfcd19a04c967c1c6737bfe6/specification/servicenetworking/ServiceNetworking.Management/tspconfig.yaml#L1-L8